core/src/main/java/org/elasticsearch/index/query/QueryBuilders.java

```diff
@@ -277,8 +277,8 @@ public static SimpleQueryStringBuilder simpleQueryStringQuery(String queryString
      * Unlike the "NOT" clause, this still selects documents that contain undesirable terms,
      * but reduces their overall score:
      */
-    public static BoostingQueryBuilder boostingQuery() {
-        return new BoostingQueryBuilder();
+    public static BoostingQueryBuilder boostingQuery(QueryBuilder positiveQuery, QueryBuilder negativeQuery) {
```

Comment:
this is breaking for the java api, we can make it but then we should start keeping track of breaking changes somewhere, I think we managed not to break anything till now